package.json

```diff
@@ -13,14 +13,17 @@
     "set-value": "^3.0.2"
   },
   "devDependencies": {
+    "cypress": "^6.4.0",
     "eslint": "^7.13.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.22.1",
     "jest": "^26.6.3",
     "webpack-dev-server": "^3.11.0"
   },
   "scripts": {
-    "test": "jest --passWithNoTests"
+    "test": "jest --passWithNoTests",
+    "cypress:open": "cypress open --project ./spec",
+    "cypress:run": "cypress run --project ./spec"
```

Comment:
What's the difference between the two?

Reply:
open opens the interface for running tests during development like I showed yesterday, run runs all the tests in the command line (and is basically for ci)

Reply:
run is headless, open is interactive

Reply:
Seems like resolved